Cargo.toml

```diff
@@ -67,6 +67,7 @@ subspace-core-primitives = { opt-level = 3 }
 subspace-erasure-coding = { opt-level = 3 }
 subspace-farmer-components = { opt-level = 3 }
 subspace-kzg = { opt-level = 3 }
+subspace-logging = { opt-level = 3 }
```

Comment:
Just copy & paste, no idea yet why

Reply:
Compiling with optimisation in dev builds probably isn’t needed, because we only call the init function once.